atxsdata/data.go

```diff
@@ -0,0 +1,180 @@
+package atxsdata
+
+import (
+	"sync"
+
+	"go.uber.org/atomic"
```

Comment:
What's the benefit of using uber's `atomic.Uint32` compared to `sync/atomic.Uint32`?
```suggestion
	"sync"
	"sync/atomic"
```

Reply:
i missed when stdlib added atomic.Uint32. i remember it only had functions to work with *uint32